libs/filabridge/include/private/filament/UibStructs.h

```diff
@@ -227,6 +226,14 @@ struct PerRenderableUibBone { // NOLINT(cppcoreguidelines-pro-type-member-init)
 static_assert(CONFIG_MAX_BONE_COUNT * sizeof(PerRenderableUibBone) <= 16384,
         "PerRenderableUibBone exceed max UBO size");
 
+struct alignas(16) PerRenderableMorphingUib {
+    static constexpr utils::StaticString _name{ "MorphingUniforms" };
+    uint32_t count;
```

Comment:
Actually I think the count shouldn't be here, it should be in the `PerRenderableUib`

Reply:
Ok. It's moved to `PerRenderableUib`.